riemann/tests/tx/test_tx.py

```diff
@@ -1253,17 +1332,17 @@ def test_init_errors(self):
         self.assertIn('Too many inputs or outputs. Stop that.',
                       str(context.exception))
 
-        # with self.assertRaises(ValueError) as context:
-        #     tx.DecredTx(
-        #         version=self.version,
-        #         tx_ins=[self.tx_in] * 2,
-        #         tx_outs=[self.tx_out],
-        #         lock_time=self.lock_time,
-        #         expiry=self.expiry,
-        #         tx_witnesses=[self.witness])
-        #
-        # self.assertIn('Witness and TxIn lists must be same length. ',
-        #               str(context.exception))
+        with self.assertRaises(ValueError) as context:
```

Comment:
Can we improve this with an `attr_assert` function?

Reply:
Elegant, on it